backend/src/api/ressources/copies/index.js

```diff
@@ -0,0 +1,194 @@
+const { Router } = require('express');
+
+const router = new Router();
+const copyController = require('./controller.js');
+
+/**
+ * @api {get} /copy/:examId Get all copies related to an exam
```

Comment:
You miss the `@apiParam` annotation (eg. https://github.com/EBM-2017-2018/OKLM/blob/52faf380641c8faa21cc4fb5c8c2a258e6fd0a25/backend/src/api/resources/categories/index.js#L41)

Otherwise, great documentation! :+1: